crates/sui-core/src/authority_active/gossip/mod.rs

```diff
@@ -205,7 +206,7 @@ async fn follower_process<A, Handler: DigestHandler<A> + Clone>(
             peer_names.insert(name);
             let local_active_ref_copy = local_active.clone();
             let handler_clone = handler.clone();
-            gossip_tasks.push(async move {
+            gossip_tasks.push(monitored_future!(async move {
```

Comment:
What is monitored here exactly?

Reply:
its keeping track of the number of live futures spawned at each callsite. See sui-metrics/src/lib.rs